### PR TITLE
Turn off datepicker autocomplete 

### LINKED ISSUE #2066: Disable browser autocomplete on deadline text field
Screen Shot 2019-07-22 at 3 17 01 PM

The browser autocomplete is blocking the calendar. Let's :fire: the autocomplete on this field.

### PR DESCRIPTION
**Pull request by GQuirino**
Remove autocomplete from deadline input datepicker.

Fixes #2066

### CODE REVIEW

**Comment by GQuirino** [comment]
@d12 here is my PR about #2066

File: app/views/assignments/_assignment_form_options.html.erb [hunk 1]
```diff
@@ -67,7 +67,7 @@
 
 <dl class="<%= view.form_class_for(:deadline) %>">
   <dt><label>Deadline <small>(optional)</small></label></dt>
-  <dd><%= f.text_field :deadline, class: 'jquery-datetimepicker form-control input-contrast input-block', value: DeadlineFormatHelper.convert(f.object.deadline) %></dd>
+  <dd><%= f.text_field :deadline, class: 'jquery-datetimepicker form-control input-contrast input-block', value: DeadlineFormatHelper.convert(f.object.deadline), :autocomplete => :off %></dd>
```
Comment (BenEmdon) [inline app/views/assignments/_assignment_form_options.html.erb]: We prefer to use the Ruby 1.9 hash literal syntax when your hash keys are symbols. ([style](https://github.com/rubocop-hq/ruby-style-guide#hash-literals))
```suggestion
  <dd><%= f.text_field :deadline, class: 'jquery-datetimepicker form-control input-contrast input-block', value: DeadlineFormatHelper.convert(f.object.deadline), autocomplete: :off %></dd>
```

**Comment by stephaniegiang** [comment]
Thanks so much for contributing!  Let us know if you need any help with the merge conflicts 😄 

Also updated your description so that it automatically closes the issue once this is merged.

**Comment by GQuirino** [comment]
i resolved the conflicts @stephaniegiang

**Comment by stephaniegiang** [comment]
Working on getting this PR merged

**Comment by d12** [comment]
Thank you so much @GQuirino :)

**Pull request merged**